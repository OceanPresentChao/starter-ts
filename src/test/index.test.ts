import { sum } from '../index';
describe("test", () => {
    test('adds 1 + 2 to equal 3', () => {
        expect(sum(5, 52)).toMatchSnapshot(`test`)
    });
})