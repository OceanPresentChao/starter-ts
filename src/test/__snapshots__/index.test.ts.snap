// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test adds 1 + 2 to equal 3: test 1`] = `57`;
